conda_smithy/configure_feedstock.py

```diff
@@ -1138,7 +1138,7 @@ def _azure_specific_setup(jinja_env, forge_config, forge_dir, platform):
             continue
         config_rendered = OrderedDict({
             "CONFIG": data["config_name"],
-            "UPLOAD_PACKAGES": str(data["upload"]),
```

Comment:
This was specifically added by me in this PR to make sure the formatting didn't change from `True` to `true`. Is this not the cases anymore?

Reply:
Oh, I thought `true` is expected for yaml files?

Reply:
We have case-sensitive comparisons elsewhere https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/templates/build_steps.sh.tmpl#L45

Reply:
I see, fixed.